src/licensedcode/data/rules/eupl-1.0_14.RULE

```diff
@@ -0,0 +1,7 @@
+---
+license_expression: eupl-1.0
+is_license_reference: yes
+relevance: 100
+---
+
+European Union Public Licence V.1.0 
```

Comment:
```suggestion
{{European Union Public Licence V.1.0}} 
```
Can you add required phrases like this to all the rules as applicable? Do the same for your other PRs too
This makes sure we are not matching this rule to similar texts partially as false positives.
Eventually these will be required phrase rules in https://github.com/aboutcode-org/scancode-toolkit/pull/3924